tensorflow/python/keras/saving/model_config.py

```diff
@@ -31,9 +31,15 @@
 # pylint: enable=g-import-not-at-top
 
 
-@keras_export('keras.models.model_from_config')
+@keras_export('keras.models.from_config')
 def model_from_config(config, custom_objects=None):
   """Instantiates a Keras model from its config.
+ 
+  Usage:
+  # for a Functional API model
+  >>> tf.keras.Model().from_config(model.get_config())
+  # for a Sequential model
+  >>> tf.keras.Sequential().from_config(model.get_config())
```

Comment:
Using ">>>" indicates that this should be a testable code snippet, which starts from a blank slate (with tensorflow imported). This would raise an error since `model` is not defined. See https://www.tensorflow.org/community/contribute/docs_ref for details about the tests and proper formatting.

Another option is adding backticks, e.g.
```
# for a Functional API model
tf.keras.Model().from_config(model.get_config())

# for a Sequential model
tf.keras.Sequential().from_config(model.get_config())
```

Reply:
Done.

Reply:
github ate my backticks, but can you add them to the comment?
```
\```
# for a Functional API model
tf.keras.Model().from_config(model.get_config())

# for a Sequential model
tf.keras.Sequential().from_config(model.get_config())
\```
```
(remove the back slashes)

Reply:
sure!